pkg/web_app/lib/src/gtag_js.dart

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2021, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+@JS('window')
+library gtag_js;
+
+import 'package:js/js.dart';
+
+@JS('gtag')
+external dynamic _gtag(String type, String action, Map<String, dynamic> data);
+
+void gtagEvent(String action, Map<String, dynamic> data) =>
+    _gtag('event', action, data);
```

Comment:
I don't know js binding so well, but don't have to call:
https://api.dart.dev/beta/2.12.0-133.2.beta/dart-js_util/jsify.html

Or create anonymous objects... at-least if we have nested objects?

Reply:
We could also use named parameters to make this easier to use... less powerful, but we probably won't use all features anyways :D